projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1045,27 +1045,40 @@ public static function notice( $message, $status = 'info', $classes = '' ) {
 	 * @param string $slug Slug of the block.
 	 */
 	public static function set_availability_for_plan( $slug ) {
-		$is_available = true;
-		$plan         = '';
-		$slug         = self::remove_extension_prefix( $slug );
-
-		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			if ( ! class_exists( 'Store_Product_List' ) ) {
-				require WP_CONTENT_DIR . '/admin-plugins/wpcom-billing/store-product-list.php';
+		$is_available   = true;
+		$plan           = '';
+		$slug           = self::remove_extension_prefix( $slug );
+		$features_data  = array();
+		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
```

Comment:
Since we do this check somewhere else in the class, maybe we could extract this?

Reply:
do you mean adding a static method to this class, for instance?

Reply:
Yeah. It does feel a bit cleaner and may be useful for other changes in the future, but let me know if you think that'd be overkill!

Reply:
it isn't at all. Let me update the code.

Reply:
Thinking about this again... I'm not sure whether adding this as a static method will bring us some benefit. For instance, we use `defined( 'IS_ATOMIC ) && IS_ATOMIC;` in the first scripting file jetpack.php. If we use some methods, then we'll have to require this Class.
Probably we could define some helper functions but does it worth it? I don't think so imho.

Reply:
There are lots of uses of `defined( 'IS_WPCOM' ) && IS_WPCOM;` throughout the codebase, but it is also sometimes pulled out to [a helper function like here](https://github.com/Automattic/jetpack/blob/f8969425fdf4fe2aa08a797ea86d321238436ba4/projects/plugins/jetpack/_inc/lib/class-jetpack-mapbox-helper.php#L78-L80). It's probably just a question of style, but if we're using it twice in the same class it might be useful to add a function.

Should we create a global `is_wpcom` or `is_simple_site` function, so any changes in this logic will be in one place?

Reply:
let me deal with this in a follow-up task, folks. I've removed the IS_ATOMIC part here 67f0b5a8a06b38af24df36f9b62a9a24ffe05df9